lockbox-ios/View/SettingListView.swift

```diff
@@ -83,6 +83,7 @@ extension SettingListView {
         if let presenter = self.presenter {
             self.tableView.rx.itemSelected
                     .map { path -> SettingRouteAction? in
+                        self.tableView.deselectRow(at: path, animated: true)
```

Comment:
Ideally, this would be done in a way that lets us set the background color for the `highlighted` state. You don't need to manually deselect cells; rather, you can set the `selectionStyle` for the tableview to `None`, and add the `tableViewCellHighlighted` color (from `Constants.swift`) for the highlighted state.

Reply:
Ah thanks! I'll fix that up at lunch.